python/GafferSceneTest/SceneTestCase.py

```diff
@@ -406,3 +406,17 @@ def __formatPaths( self, path1, path2 ) :
 			return "\"{0}\"".format( path1 )
 		else :
 			return "\"{0}\" and \"{1}\"".format( path1, path2 )
+
+	@staticmethod
+	def assertScenesRenderSame( plugA, plugB, expandProcedurals = False, ignoreLinks = False ):
+		rendererA = GafferScene.Private.IECoreScenePreview.CapturingRenderer( GafferScene.Private.IECoreScenePreview.Renderer.RenderType.Batch )
+		controllerA = GafferScene.RenderController( plugA, Gaffer.Context(), rendererA )
+		controllerA.setMinimumExpansionDepth( 1024 )
+		controllerA.update()
+
+		rendererB = GafferScene.Private.IECoreScenePreview.CapturingRenderer( GafferScene.Private.IECoreScenePreview.Renderer.RenderType.Batch )
+		controllerB = GafferScene.RenderController( plugB, Gaffer.Context(), rendererB )
+		controllerB.setMinimumExpansionDepth( 1024 )
+		controllerB.update()
+
```

Comment:
Might be worth adding a todo about also testing with a `Render` node, since that is a separate code path, but should give us the same result as the RenderController?